packages/entrykit/src/icons/ArrowLeftIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { IconSVG, Props } from "./IconSVG";
+
+export function ArrowLeftIcon(props: Props) {
+  return (
+    <IconSVG strokeWidth="2" stroke="currentColor" {...props}>
```

Comment:
```suggestion
    <IconSVG {...props}>
```
`<path>` specifies these so I don't think these are needed

Reply:
indeed, good catch, removed